easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-10.3.0-largefiles.eb

```
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenSlide'
+version = '3.4.1'
+versionsuffix = '-largefiles'
+
+homepage = 'https://openslide.org/'
+description = """OpenSlide is a C library that provides a simple interface to
+read whole-slide images (also known as virtual slides)."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_large_file_support.patch']
+checksums = [
+    # v3.4.1.tar.gz
+    'a5d869916e370125421535dcce778b2ba625dc50d920aa4ca93bbaaa6a7b470c',
+    # %(name)s-%(version_major_minor)s.1_large_file_support.patch
+    'cb618053f4ae6c3ce37d1b8b0e4ef7c55fd17378776d13be4aa4efab91706b8c',
+]
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('M4', '1.4.18'),
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.36.1'),
+]
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('LibTIFF', '4.2.0'),
+    ('OpenJPEG', '2.4.0'),
+    ('libxml2', '2.9.10'),
+    ('SQLite', '3.35.4'),
+    ('cairo', '1.16.0'),
+    ('Gdk-Pixbuf', '2.42.6'),
+]
+
+preconfigopts = "autoreconf -f -i && "
+
+github_account = '%(namelower)s'
```

`github_account` is related to the source, so it's best to move this to the other source variables.
```suggestion
github_account = '%(namelower)s'
source_urls = [GITHUB_SOURCE]
sources = ['v%(version)s.tar.gz']
patches = ['%(name)s-%(version)s_large_file_support.patch']
checksums = [
    # v3.4.1.tar.gz
    'a5d869916e370125421535dcce778b2ba625dc50d920aa4ca93bbaaa6a7b470c',
    # %(name)s-%(version_major_minor)s.1_large_file_support.patch
    'cb618053f4ae6c3ce37d1b8b0e4ef7c55fd17378776d13be4aa4efab91706b8c',
]

builddependencies = [
    ('Autotools', '20210128'),
    ('M4', '1.4.18'),
    ('pkg-config', '0.29.2'),
    ('binutils', '2.36.1'),
]

dependencies = [
    ('zlib', '1.2.11'),
    ('libpng', '1.6.37'),
    ('libjpeg-turbo', '2.0.6'),
    ('LibTIFF', '4.2.0'),
    ('OpenJPEG', '2.4.0'),
    ('libxml2', '2.9.10'),
    ('SQLite', '3.35.4'),
    ('cairo', '1.16.0'),
    ('Gdk-Pixbuf', '2.42.6'),
]

preconfigopts = "autoreconf -f -i && "
```